config/config.go

```diff
@@ -299,7 +299,7 @@ func migrateConfigDir() error {
 			// write back
 			writeErr := ioutil.WriteFile(theKubeConfigPath, []byte(newConfig), stat.Mode())
 			if writeErr != nil {
-				return fmt.Errorf("Could not overwrite kubectl config file. %s", writeErr.Error())
+				return errors.New("could not overwrite kubectl config file")
```

Comment:
I'd say do it like `return fmt.Errorf("could not overwrite kubectl config file: %s", writeErr.Error())`, unless you don't care about writeErr message.

BTW in almost all cases error variable is called `err` and reused.